### PR TITLE
[RSDK-8581] Stop using HW PWM on Jetson/Orin boards

### PR DESCRIPTION
**Pull request by penguinland**
Both the Orin and the Orin Nano can have a significant delay (hundreds of milliseconds?) when changing the output from a hardware PWM chip. This is now exacerbated, since setting those pins high or low now means setting the PWM duty cycle to 100% or 0%, rather than using the GPIO capabilities. 

Should this be generalized to the entire Jetson line? We don't test regularly on the others, so I don't know if the issue affects them, too. and in practice, no one is using these other boards, so it's not a big deal either way. I've removed all their uses, but did it in a separate commit so it's easy to undo just the generalization if we want.

Should the PWM stuff be commented out or removed entirely? We might want to put it back later (see RSDK-8357), but it's easy to restore either way. If they should be removed entirely, the code looks cleaner, but undoing the change requires reverting this PR because it's a pain to recreate the code without reverting.

Will test when there is a corresponding PR in the board canaries repo to remove the sleeps...

### CODE REVIEW

**Comment by penguinland** [comment]
Tried on an Orin AGX: along with https://github.com/viamrobotics/board_canaries/pull/40 (which removes the sleeps from the tests that accommodated the PWM delays), the canary tests have passed 15 times in a row.

Tried on an Orin Nano: same results, 15 times in a row.

File: components/board/jetson/data.go [hunk 1]
```diff
@@ -94,8 +94,8 @@ var jetsonNanoPins = []genericlinux.PinDefinition{
 	{Name: "31", DeviceName: "gpiochip0", LineNumber: 200, PwmChipSysfsDir: "", PwmID: -1},
 	// Older versions of L4T have a DT bug which instantiates a bogus device
 	// which prevents this library from using this PWM channel.
-	{Name: "32", DeviceName: "gpiochip0", LineNumber: 168, PwmChipSysfsDir: "7000a000.pwm", PwmID: 0},
-	{Name: "33", DeviceName: "gpiochip0", LineNumber: 38, PwmChipSysfsDir: "7000a000.pwm", PwmID: 2},
+	{Name: "32", DeviceName: "gpiochip0", LineNumber: 168, PwmChipSysfsDir: "", PwmID: -1}, // "7000a000.pwm", PwmID: 0},
+	{Name: "33", DeviceName: "gpiochip0", LineNumber: 38, PwmChipSysfsDir: "", PwmID: -1},  // "7000a000.pwm", PwmID: 2},
```
Comment (martha-johnston) [inline components/board/jetson/data.go:98]: I'm not at all familiar with this type of data file, but I notice that all the other PwmIDs changed from 0 to -1. why does this one change from 2 to -1?

Comment (penguinland) [reply components/board/jetson/data.go:98]: -1 means "this pin has no hardware PWM associated with it," so removing HW PWM is done by setting them all to -1.

If the PWM ID is _not_ -1, it's the line on the PWM chip that connects to this pin. Most chips NVidia uses have 1 line per chip, and they add extra chips to get extra PWM capabilities. However, some boards use a PWM chip with 4 lines coming out of it: line 0 is attached to physical pin 32, line 1 IIRC is the cooling fan on the CPU's heatsink, line 2 is physical pin 33, and line 3 is... something else I don't remember.

**Comment by martha-johnston** [comment]
I don't fully understand this so I'm not sure I'm the best person to review, but I think your explanation makes sense and if all the tests are passing that is fine by me

**Review by martha-johnston — APPROVED** [review]

**Comment by penguinland** [comment]
Yeah, if this were a difficult change I'd want a review from someone more familiar with this. but IMO the PR is pretty straightforward, so I think your review is alright. and now you know a little more for next time!

**Pull request merged**